### ISSUE TITLE
Regarding zero shot training setting

### ISSUE THREAD

**Issue by vidit98** [MendelXu/zsseg.baseline#6]
Thanks for the great work and the code. I had some doubts regarding the zero-shot training setting. If I understand correctly, in zero-shot setting only the base classes are used for training and the novel classes should be ignored maybe by setting them as ignore labels.

I didn't find any code to handle the novel classes. I was expecting that in the file `zsseg.baseline/mask_former/data/dataset_mappers/mask_former_semantic_dataset_mapper.py` or `zsseg.baseline/mask_former/zero_shot_mask_former_model.py` there should be a provision to ignore the novel classes while training. 

I can see that you only pass the names of base classes [here](https://github.com/MendelXu/zsseg.baseline/blob/bb6504aa8b573de1afa7f58c8e5d40de524f0a58/mask_former/data/datasets/register_voc_seg.py#L121) which are used in text prompts but I don't see any code to modify the groundtruth segmentation masks to ignore the novel classes.

Please let me know if I am missing something or misunderstanding something.
